website/docs/r/datapipeline.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "aws"
+page_title: "AWS: aws_datapipeline"
+sidebar_current: "docs-aws-resource-datapipeline"
+description: |-
+  Provides a AWS DataPipeline.
+---
+
+# aws_datapipeline
```

Comment:
Among other renaming in here, we have since added `Resource: ` in the top headers for resources. 👍 

```suggestion
# Resource: aws_datapipeline_pipeline
```

Reply:
I fixed.